pkg/apis/kubermatic/v1/cluster.go

```diff
@@ -396,22 +410,22 @@ type ClusterStatus struct {
 	// +optional
 	Versions ClusterVersionsStatus `json:"versions,omitempty"`
 
-	// UserName contains the name of the owner of this cluster
+	// UserName contains the name of the owner of this cluster.
 	// +optional
 	UserName string `json:"userName,omitempty"`
```

Comment:
Can you please mark this as deprecated as well? It is IIRC not used anywhere and never set by anyone anymore. It got added in The Before Times (#523) without any context or usecase.

The UserEmail is still used for one single thing (binding the email to the cluster-admin ClusterRole inside a usercluster), but that is also of vanishingly importance with GitOps (if you use GitOps, you will most likely setup RBAC in your cluster in your own way, instead of relying on the ClusterStatus magic for one very specific usecase). So that's why I would say to _not_ mark the UserEmail as deprecated.

Reply:
Done.